pkg/resources/manifests/deploy.yaml

```diff
@@ -102,6 +102,10 @@ spec:
         # (For reference, see: https://github.com/inspektor-gadget/inspektor-gadget/runs/3966318270?check_suite_focus=true#step:20:221)
         container.apparmor.security.beta.kubernetes.io/gadget: "unconfined"
         inspektor-gadget.kinvolk.io/option-hook-mode: "auto"
+        # keep aligned with values in pkg/operators/prometheus/prometheus.go
```

Comment:
Nice. Probably we should also document how to disable scrapping with something:

```bash
$ kubectl annotate daemonsets -l k8s-app=gadget -n gadget prometheus.io/scrape=false --overwrite
$ kubectl annotate pod -l k8s-app=gadget -n gadget prometheus.io/scrape=false
```

Reply:
Or make it a deploy flag? `.. deploy --disable-prometheus-autodiscovery`?